docs/DeveloperGuide.md

```diff
@@ -257,27 +257,55 @@ _{Explain here how the data archiving feature will be implemented}_
-| `* * *`  | user                                       | add a new person               |                                                                        |
-| `* * *`  | user                                       | delete a person                | remove entries that I no longer need                                   |
-| `* * *`  | user                                       | find a person by name          | locate details of persons without having to go through the entire list |
-| `* *`    | user                                       | hide private contact details   | minimize chance of someone else seeing them by accident                |
-| `*`      | user with many persons in the address book | sort persons by name           | locate a person easily                                                 |
+| Priority | As a …                | I can …                                                                                                  | So that …                                                                                                                                       |
+|----------|-----------------------|----------------------------------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------|
+| `*`      | medical administrator | export a current week’s schedule for nurses so that                                                      | they do not need to contact me for their schedules repeatedly                                                                                   |
+| `* * *`  | medical administrator | quickly add the name, age, address and contact of the patients that require home visits so that          | retrieve them later                                                                                                                             |
+| `* * *`  | medical administrator | quickly add the name, age, address and contact of the nurse/volunteers                                   | retrieve them later for matching                                                                                                                |
+| `* * *`  | medical administrator | add the health conditions (eg heart disease, asthma, paralysed etc) of the patients                      | use this information to match the patient with the respective nurse that has experience in dealing with such disease                            |
+| `* *`    | medical administrator | add a list of diseases that the nurse dealt with before                                                  | give them the appropriate patient for a home visit                                                                                              |
```

Comment:
This 2 maybe say "add the health conditions.... as tags for the patients" . Similar for nurse.